<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sorts</class>
 <widget class="QMainWindow" name="Sorts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listRandom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>161</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRandom">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuick">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>QuickSort</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listQuick">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>161</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMerge">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MergSort</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listMerge">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>161</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBin">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>BinSort</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listBin">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>161</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHeap">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>HeapSort</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listHeap">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>161</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBubble">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>BubbleSort</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listBubble">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>90</y>
      <width>161</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="txtQuick">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtMerge">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtBin">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtHeap">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>60</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtBubble">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>60</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKenyStev">
    <property name="title">
     <string>KenyStev</string>
    </property>
   </widget>
   <addaction name="menuKenyStev"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
